aces/Loris
# [BVL Feedback] Restore BVL Feedback link at the instrument level

## Brief summary of changes

This PR passes the actual instrument/test name i.e. mri_parameter_form instead of the string "instruments" ($page->name is "instruments because it's the name of the module) to `UserPageDecorationMiddleware` so that `\NDB_BVL_Feedback::bvlFeedbackPossible()` can be called with the correct testname.

#### Testing instructions (if applicable)

1. Go to any instrument page on 23.0-release branch. Notice that the BVL feedback icon does not appear on the navigation bar.
2. On this branch, notice that the icon is restored.

#### Link(s) to related issue(s)

* Resolves https://github.com/aces/Loris/issues/6259
